.github/actions/upgrade-indexer/common.sh

```diff
@@ -0,0 +1,121 @@
+#!/bin/bash
+FILES_OLD="/usr/share/wazuh-indexer/plugins/opensearch-security/securityconfig"
+FILES_NEW="/etc/wazuh-indexer/opensearch-security"
+declare -A files_old
+declare -A files_new
+PACKAGE_NAME="$1"
+MAJOR_MINOR_RELEASE=$(($2))
+REFERENCE_VERSION=43
+
+EQUAL=true
+
+# Check the system to differ between DEB and RPM
+function check_system() {
+
+    if [ -n "$(command -v yum)" ]; then
+        sys_type="rpm"
+    elif [ -n "$(command -v apt-get)" ]; then
+        sys_type="deb"
+    else
+        echo "Error: could not detect the system"
```

Comment:
If the string is not going to be concatenated, the use of punctuation symbols should be considered, such as the final period.

echo "Error: could not detect the system`.`"

Reply:
Done.